jenkins_jobs/modules/triggers.py

```diff
@@ -155,6 +155,7 @@ def build_gerrit_triggers(xml_parent, data):
         "patchset-uploaded-event": "PluginPatchsetCreatedEvent",
         "patchset-created-event": "PluginPatchsetCreatedEvent",
         "ref-updated-event": "PluginRefUpdatedEvent",
+        "topic-changed-event": "PluginTopicChangedEvent",
```

Comment:
Looks like the keys were meant to be ordered alphabetically, could you please fix that? (and also the order of "private-state-changed-event")